### PR TITLE
feat: create a new tweet

### PR DESCRIPTION
**Pull request by Weii-Dev9**
## Overview
- Reference card [#16](https://trello.com/c/vQJFAY9I/16-3当前用户可以编辑发送朋友圈4)

Why was this PR created / what was done?

- create a new tweet

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
1. [x] Makes your PR pass tests?
2. [x] PR attached to trello card
3. [x] Trello card moved into 'Review'

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -16,7 +16,8 @@ android {
         versionCode 1
         versionName "1.0"
 
-        testInstrumentationRunner "com.thoughtworks.androidtrain.helloworld.HiltTestRunner"
+        //testInstrumentationRunner "com.thoughtworks.androidtrain.helloworld.HiltTestRunner"
```
Comment (yunlongwen) [inline app/build.gradle]: 如果代码需要注释并保留，建议说明 原因和 情况(是个TODO项？)。
如果不需要保留，请删除。

File: app/src/androidTest/java/com/thoughtworks/androidtrain/helloworld/dao/TweetInsertTest.kt [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        senderEntity.avatar = "own_pic"
+        senderEntity.username = "Weii"
+
+        senderDao.insert(senderEntity)
+
+        Assert.assertEquals(1, senderDao.getAll().size)
+    }
+
+    @Test
+    fun insertTweetEntity_test() = runTest {
```
Comment (yunlongwen) [inline app/src/androidTest/java/com/thoughtworks/androidtrain/helloworld/dao/TweetInsertTest.kt]: 方法命名方式 保持一致。

File: app/build.gradle (hunk 1 above)
Comment (Weii-Dev9) [reply app/build.gradle]: 已保留HiltTestRunner，HiltTestRunner可以兼容JUnitRunner

File: app/src/androidTest/java/com/thoughtworks/androidtrain/helloworld/dao/TweetInsertTest.kt (hunk 2 above)
Comment (Weii-Dev9) [reply app/src/androidTest/java/com/thoughtworks/androidtrain/helloworld/dao/TweetInsertTest.kt]: 已修改

**Review by yunlongwen — APPROVED** [review]

**Pull request merged**